functions/hierarchyCheck.js

```diff
@@ -0,0 +1,44 @@
+/**
+Check if user is higher in guild's role hierarchy
+...Basically an easy way to stop mods from performing actions on admins, etc
+
+    Usage:
+        - client scope
+        - executor is the user object of the user executing the moderation command
+        - target is the user object of the user the moderation command is affecting
+        - guild is the guild object of the guild the command is being executed in
+    Returns:
+        - boolean true if executor is higher
+        - boolean false if executor is lower or equal
+        - boolean false if either user isn't in the guild
+
+        **/
+
+const func = async (client, executor, target, guild = null) => {
+  try {
```

Comment:
try/catch isn't needed, it'll make the function always resolve even if we don't want it to. This function is very unlikely to fail, let it be catch by the developer in their commands.